app/src/main/java/com/kuit/kuit6android/ui/home/component/MidBanner.kt

```diff
@@ -0,0 +1,98 @@
+package com.kuit.kuit6android.ui.home.component
+
+import androidx.compose.foundation.background
+import androidx.compose.foundation.clickable
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.size
+import androidx.compose.foundation.layout.width
+import androidx.compose.foundation.layout.wrapContentSize
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.text.style.TextAlign
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
+import com.kuit.kuit6android.ui.home.component.MidBanner
+
+@Composable
+fun MidBanner(modifier: Modifier = Modifier) {
+    Row(
+        modifier = Modifier
+//            .padding(vertical = 20.dp)
+//            .size(
+//                width = 306.dp,
+//                height = 77.dp
+//            )
+            .width(306.dp)
```

Comment:
```suggestion
            .fillMaxWidth()
            .padding(horizontal = 27.dp)
```

이 부분도 상기한 이유와 같은 이유로 고정 폭 대신 화면 전체 너비를 채우고, 패딩을 주는 방식이 더 좋을 것 같습니다 !